internal/driver/webhtml.go

```diff
@@ -741,22 +744,147 @@ function viewer(baseUrl, nodes) {
 <meta charset="utf-8">
 <title>{{.Title}}</title>
 {{template "css" .}}
+<style type="text/css">
+#flathdr1, #flathdr2, #cumhdr1, #cumhdr2, #namehdr {
+  cursor: ns-resize;
+}
+</style>
 </head>
 <body>
 
 {{template "header" .}}
 
 <div id="bodycontainer">
 <table id="toptable">
-<tr><th>Flat<th>Flat%<th>Sum%<th>Cum<th>Cum%<th>Name<th>Inlined?</tr>
+<tr>
+<th id="flathdr1">Flat
+<th id="flathdr2">Flat%
+<th>Sum%
+<th id="cumhdr1">Cum
+<th id="cumhdr2">Cum%
+<th id="namehdr">Name
+<th>Inlined?</tr>
+<tbody id="rows">
 {{range $i,$e := .Top}}
   <tr id="node{{$i}}"><td>{{$e.Flat}}<td>{{$e.FlatPercent}}<td>{{$e.SumPercent}}<td>{{$e.Cum}}<td>{{$e.CumPercent}}<td>{{$e.Name}}<td>{{$e.InlineLabel}}</tr>
 {{end}}
+</tbody>
 </table>
 </div>
 
 {{template "script" .}}
-<script>viewer({{.BaseURL}}, {{.Nodes}})</script>
+<script>
+function makeSortable() {
+  // Regular expression to extract leading number and a suffix.
+  // E.g., "1.3s" => 1.3, "s"
+  let numParseRe = new RegExp("^([\\-+]?[0-9]+([.][0-9]+)?)(.*)$")
+
+  // Unit conversions (for toUnit values produced by measurement.go)
+  const units = new Map([
+    ["B", 1.0],
```

Comment:
there is no difference when ".0" is added to a number, 1 and 1.0 is exactly the same number and type in JS.

Reply:
Done.